src/ontology/edits/oeo-physical.omn

```diff
@@ -7142,6 +7151,33 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/946",
         OEO_00000350
     
     
+Class: OEO_00240018
+
+    Annotations: 
+        OEO_00110012 "gross\\_national\\_electricity\\_consumption = gross\\_electricity\\_generation + electricity\\_imports - electricity\\_exports",
```

Comment:
Is that proper LaTeX syntax? I think, it should be only single backslashes.

Reply:
![image](https://user-images.githubusercontent.com/84896056/150941300-4f84b8a0-82ca-44b3-8c16-5a94a0a189d0.png)
That´s how it looks like in Protégé and how I implemented it. 
Is one of the two backslashes automatically added in GitHub when `_` is used?

If I delete the respective backslash in Protégé, both backslashes disappear in GitHub-Desktop and probably also on GitHub.com:
![image](https://user-images.githubusercontent.com/84896056/150942202-6a46efa0-fef3-4ec1-a735-2f19313507a0.png)

Reply:
@Ludee Do you have any idea? What do you prefer?

Reply:
@Ludee, I thought you tested whether LaTeX syntax works properly in Protégé?

Reply:
Looks like my tests were not sufficient. Sorry for that.
I could reproduce the additional backslash but not the deletion of the file.

In order to fix this we decided to only use CamelCase in the formulas and avoid underscores and backslashes!
I implemented this here and will document it properly asap.